src/components/footer.astro

```diff
@@ -0,0 +1,36 @@
+<html>
+  <body>
+    <footer
+      class="flex h-20 items-center justify-between bg-gray-50 px-6 text-gray-600 dark:bg-gray-950 dark:text-gray-300"
+    >
+      <p>
+        <span class="max-sm:hidden">Copyright</span>
+        &copy; {new Date().getUTCFullYear()}
```

Comment:
there may be a caveat with this, if this code runs once, on build, the year could be out of date if a build doesn't happen in awhile.